appgate/__main__.py

```diff
@@ -291,7 +291,10 @@ def main_api_info(spec_directory: Optional[str] = None) -> None:
 
 
 def main_dump_crd(
-    stdout: bool, output_file: Optional[str], spec_directory: Optional[str] = None
+    stdout: bool,
+    output_file: Optional[str],
+    spec_directory: Optional[str] = None,
+    prefix: Optional[str] = None,
```

Comment:
Does it make sense to have it Optional, is there any case that we dont want the API to be prefixed with the version?